x-pack/test/functional/apps/ml/data_frame_analytics/feature_importance.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { DeepPartial } from '../../../../../plugins/ml/common/types/common';
+import { DataFrameAnalyticsConfig } from '../../../../../plugins/ml/public/application/data_frame_analytics/common';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  describe('jobs cloning supported by UI form', function () {
+    const testDataList: Array<{
+      suiteTitle: string;
+      archive: string;
+      indexPattern: { name: string; timeField: string };
+      job: DeepPartial<DataFrameAnalyticsConfig>;
+    }> = (() => {
+      const timestamp = Date.now();
+
+      return [
+        {
+          suiteTitle: 'binary classification job',
+          archive: 'ml/bm_classification',
+          indexPattern: { name: 'ft_bank_marketing', timeField: '@timestamp' },
+          job: {
+            id: `bm_fi_binary_${timestamp}`,
+            description:
+              "Classification job based on 'ft_bank_marketing' dataset with dependentVariable 'y' and trainingPercent '20'",
```

Comment:
This refers to a trainingPercent of `20`, but `5` is used in the test.

Reply:
Updated here https://github.com/elastic/kibana/pull/82677/commits/3327bd5918cf9d46c81d370c96604b67328a64dc